src/index.html

```diff
@@ -1250,6 +1250,189 @@ <h3>Dropdowns</h3>
       </calcite-tab>
       <calcite-tab>
 
+        <h3>Big Tree</h3>
+
+        <calcite-tree lines>
+          <calcite-tree-item>
+            <a href="/guide/developing-extensions-with-arcgis-enterprise-sdk/">Developing Extensions with ArcGIS Enterprise SDK</a>
```

Comment:
We'll need to "lock" the size of the caret somehow in css - in a narrow tree nav, this will wrap lines and currently the caret shrinks in size correspondingly which is a little weird.